apidoc/v2/openapi.yaml

```diff
@@ -3668,6 +3668,16 @@ paths:
                     content:
                       type: integer
                       description: Binary value for `is_owner`, integer for `group` (2 for Admin, 4 for User)
+                - type: object
```

Comment:
this is not needed, these fields are already described by the `users` object: https://github.com/elabftw/elabftw/blob/75c618708d9dff4dbd7573945ccee41517222009/apidoc/v2/openapi.yaml#L3617